src/mongo/grammar/visitors.ts

```diff
@@ -63,8 +63,8 @@ export class MongoVisitor<T> implements mongoVisitor<T> {
 		return this.defaultResult(node);
 	}
 
-	visitErrorNode(node: ErrorNode): T {
-		throw new Error(`Error near line ${node._symbol.line}, column ${node.symbol.charPositionInLine + 1}, text '${node.text}'. Please check syntax.`);
+	visitErrorNode(_node: ErrorNode): T {
+		return null;
```

Comment:
Don't you just not implement visitErrorNode?

Reply:
This was mainly a reversion to the previous code. 
I currently see both the classes extending it (completionItemProvider and mongo/commands) to have implemented it. 

I see this as a fall back in case the next person is building out a third use case (a function returning nothing won't throw a `member not found` error). I don't feel strongly about this though.